docs/reference/server-configuration.md

```diff
@@ -680,7 +790,7 @@ without wastefully attempting to open connections to it (which can be slow & res
 
 ## `ksql.query.pull.max.allowed.offset.lag`
 
-**Per query:** yes
+**Per query:** yes (for {{ site.ccloud }}, settable with ALTER SYSTEM only)
```

Comment:
@rodesai same here, I'm skeptical but given that it explicitly mentions using the `SET` command in the docs for this one at least I'm inclined to say it actually is query level

Reply:
@rodesai, fyi this comment was in a collapsed section, and I almost missed it.

Reply:
Same here, KQE says intentionally query-level after all

```suggestion
**Per query:** yes
```